index.html

```diff
@@ -5,14 +5,21 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
KINDLY IMPLEMENT THESE CHANGES:
- Fill the information missing on your README File:
![Screen Shot 2021-09-15 at 16 19 38](https://user-images.githubusercontent.com/22258274/133511016-cb9cad31-8139-4e76-8d12-2c68daa33c4f.png)

Reply:
Sorry please what the information missing in my README file 
And I think that will not reject my code because all requirements in  [Awesome books: style core elements
](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m3_style_core_elements.md) we do it